# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: parser/html/moz.build
# Commit: 9a8abd87cc79
# Full Hash: 9a8abd87cc7935f29b94248c1a6f8203faa14403
# Author: Henri Sivonen <hsivonen@hsivonen.fi>
# Date: 2021-12-08 21:43:06
# Regressor Bug: 1701828
# File Overlap Count: 10
# Description:
#   Bug 1701828 - meta charset rewrite. r=smaug
#   
#   Implements https://github.com/whatwg/html/issues/6962 . Improves performance
#   when <meta charset> occurs in head but after the first kilobyte and aligns
#   behavior better with WebKit and Blink.
# ==============================================================================

diff --git a/parser/html/moz.build b/parser/html/moz.build
--- a/parser/html/moz.build
+++ b/parser/html/moz.build
@@ -18,18 +18,16 @@ EXPORTS += [
     "nsHtml5AttributeName.h",
     "nsHtml5ByteReadable.h",
     "nsHtml5ContentCreatorFunction.h",
     "nsHtml5DependentUTF16Buffer.h",
     "nsHtml5DocumentBuilder.h",
     "nsHtml5DocumentMode.h",
     "nsHtml5HtmlAttributes.h",
     "nsHtml5Macros.h",
-    "nsHtml5MetaScanner.h",
-    "nsHtml5MetaScannerHSupplement.h",
     "nsHtml5Module.h",
     "nsHtml5NamedCharacters.h",
     "nsHtml5NamedCharactersAccel.h",
     "nsHtml5OplessBuilder.h",
     "nsHtml5OwningUTF16Buffer.h",
     "nsHtml5Parser.h",
     "nsHtml5PlainTextUtils.h",
     "nsHtml5Portability.h",
@@ -55,17 +53,16 @@ EXPORTS += [
 UNIFIED_SOURCES += [
     "nsHtml5AtomTable.cpp",
     "nsHtml5AttributeName.cpp",
     "nsHtml5DependentUTF16Buffer.cpp",
     "nsHtml5DocumentBuilder.cpp",
     "nsHtml5ElementName.cpp",
     "nsHtml5Highlighter.cpp",
     "nsHtml5HtmlAttributes.cpp",
-    "nsHtml5MetaScanner.cpp",
     "nsHtml5Module.cpp",
     "nsHtml5NamedCharacters.cpp",
     "nsHtml5NamedCharactersAccel.cpp",
     "nsHtml5OplessBuilder.cpp",
     "nsHtml5OwningUTF16Buffer.cpp",
     "nsHtml5Parser.cpp",
     "nsHtml5PlainTextUtils.cpp",
     "nsHtml5Portability.cpp",